Doc/library/faulthandler.rst

```diff
@@ -106,8 +108,8 @@ Dumping the tracebacks after a timeout
 
    Dump the tracebacks of all threads, after a timeout of *timeout* seconds, or
    every *timeout* seconds if *repeat* is ``True``.  If *exit* is ``True``, call
-   :c:func:`_exit` with status=1 after dumping the tracebacks.  (Note
-   :c:func:`_exit` exits the process immediately, which means it doesn't do any
+   :c:func:`!_exit` with status=1 after dumping the tracebacks.  (Note
+   :c:func:`!_exit` exits the process immediately, which means it doesn't do any
```

Comment:
Is this the same as https://docs.python.org/3/library/os.html#os._exit? Should this link to that, maybe?

Reply:
I don't think so, it's calling the C one from `unistd.h`:

https://github.com/python/cpython/blob/e0ab6424369bcd276b0dc4f537fde360bdca2f05/Modules/faulthandler.c#L11

https://github.com/python/cpython/blob/e0ab6424369bcd276b0dc4f537fde360bdca2f05/Modules/faulthandler.c#L591-L592

Reply:
I think `os._exit` is just a very thin wrapper around the C one:

https://github.com/python/cpython/blob/e0ab6424369bcd276b0dc4f537fde360bdca2f05/Modules/posixmodule.c#L6613-L6619

But yes, good point, they're not exactly the same!